bin/provision.sh

```diff
@@ -25,11 +26,13 @@ systemctl enable httpd.service
 firewall-cmd --permanent --zone=public --add-service=http
 firewall-cmd --permanent --zone=public --add-service=https
 firewall-cmd --reload
+sudo ln -sf /var/www/html/bin/translations.sh
```

Comment:
pls move translation.sh to dev-oops/local.team-opencaching.de
so we can use it with psh.phar